### ISSUE TITLE
[SF][A11y][SegmentedButton]: [<svg> elements with an img role must have an alternative text]

### ISSUE THREAD

**Issue by tandan0813** [SAP/ui5-webcomponents#10573]
### Describe the bug

Please see the isolated example.
When used SegmentedButton with icon, and then run AXE tool, there will show a accessible issue saying <svg> elements with an img role must have an alternative text

![Image](https://github.com/user-attachments/assets/7799426d-9102-4d1a-934a-7c10de27cd10)

### Isolated Example

https://stackblitz.com/edit/github-zecan1zo?file=src%2FApp.tsx

### Reproduction steps

1. Open isolated example
2.Run AXE tool

### UI5 Web Components for React Version

2.5.0

### UI5 Web Components Version

2.5.1

### Browser

Chrome

### Relevant log output

```shell

```

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by IlianaB** [comment]
Hello @SAP/ui5-webcomponents-topic-b , please have a look at this accessibility issue regarding SegmentedButtonItem with icon. Axe tool is reporting missing alternative text for svg with role="img".

**Comment by unazko** [comment]
Hello @tandan0813,

The issue is fixed in the [nightly](https://sap.github.io/ui5-webcomponents/nightly/components/SegmentedButton/).
Please re-test. The icons are made decorative as per accessibility specification.

Additionally a tooltip contextual tooltip should be provided for the icon-only `ui5-segmented-buton-item'`'s in most of the cases according to the UX guidelines.
The tooltip is take from the icon if it exists. There are many icons however without a default tooltip and the icon-only button items should have a tooltip provided at application level.

Best regards,
Boyan
